build/build.sh

```diff
@@ -155,6 +155,13 @@ elif echo "${VERSION}" | grep 'renovated'; then
     MAJOR_MINOR=renovated-$(echo "${SUB_VERSION}" | cut -d'.' -f1-2)
     INSTALL_TARGET=install
     LANGUAGES="objc,c,c++"
+elif echo "${VERSION}" | grep 'gcc-thomas-healy'; then
```

Comment:
this needs to be able the `grep 'trunk'` on line 143... I'll fix locally and commit and make sure the docs and this code are clearer!